cse-cc-import-runner-app/src/main/resources/application.yml

```diff
@@ -10,10 +10,21 @@ spring:
       reply-timeout: 2400000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
+      default:
+        binder: rabbit
+        group: cse-idcc-runner
       bindings:
+        interrupt-in-0:
+          destination: cse-interrupt
+        request-in-0:
+          destination: cse-import-idcc-request
+        request-out-0:
+          destination: cse-response
         task-status-update:
-          binder: rabbit
-          destination: task-status-updated
+          destination: cse-import-idcc-task-status-updated
+          group: cse-import-idcc-task-manager
```

Comment:
You can remove the group here, it's a production flux so it won't be used

Reply:
no, otherwise the task-manager is not abble to receive the status update. it is important to specify the correct group